client/src/components/Chart/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import _ from 'lodash';
+import { VictoryChart, VictoryLine, VictoryZoomContainer } from 'victory';
+
+const Chart = (props) => {
+  const [zoomedXDomain, setZoomedXDomain] = useState(props.entireDomain.x);
+
+  const onDomainChange = (domain) => {
+    setZoomedXDomain(domain.x);
+  };
+
+  const getData = () => {
```

Comment:
will be nice to have a one liner explaining what is happening on in this code since it's not obvious why the 2x filtering is necessary